esmvalcore/cmor/_fixes/cmip6/fgoals_g3.py

```diff
@@ -32,13 +33,13 @@ def fix_metadata(self, cubes):
         Returns
         -------
         iris.cube.CubeList
-
         """
         cube = self.get_cube_from_list(cubes)
-        cube.coord('latitude').points[
-            cube.coord('latitude').points > 1000.0] = 0.0
-        cube.coord('longitude').points[
-            cube.coord('longitude').points > 1000.0] = 0.0
+        for coord_name in ['latitude', 'longitude']:
+            coord = cube.coord(coord_name)
+            bad_indices = coord.core_points() > 1000.0
+            coord.points = da.ma.where(bad_indices, 0.0, coord.core_points())
```

Comment:
Does this even make sense as a fix? This moves all bad points to the pole. Is the data corresponding to the bad points masked? I was not able to find any files that have the issue that is fixed here.

Reply:
no :rofl:

Reply:
No, this doesn't move it to the pole, but rather to 0°N, 0°E, which is [south of Ghana in the Atlantic Ocean](https://www.google.de/maps/place/0%C2%B000'00.0%22N+0%C2%B000'00.0%22E). Since `tos` is an ocean variable, this point should not be masked.

So no, this doesn't make sense to me at all... The funny thing is that `git blame` claims that I was the one who added this, but I can't remember anything about this 😅 Maybe we just keep it?

Reply:
You just moved it in #1092, it was introduced in #629. I found the following files on Levante that have the issue:
```
siconc_SImon_FGOALS-g3_piControl_r1i1p1f1_gn_070001-079912.nc
siconc_SImon_FGOALS-g3_piControl_r1i1p1f1_gn_080001-089912.nc
siconc_SImon_FGOALS-g3_ssp534-over_r1i1p1f1_gn_201501-210012.nc
```

Reply:
Ah, you're right. Do you maybe want to add this as a comment to the code? We might want to revisit this at some point..

Reply:
Good idea, done in the latest commit.

Reply:
Great, will merge once the tests finished